core/data/tests/test_optional_type_alias/output.py

```diff
@@ -0,0 +1,19 @@
+"""
+ Generated by typeshare 1.12.0
+"""
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+OptionalU16 = int
+
+
+OptionalU32 = int
+
+
+class FooBar(BaseModel):
+    foo: OptionalU32
+    bar: OptionalU16
```

Comment:
This looks very wrong to me– the relevant rust type is `Option<u32>`

Reply:
Fixed